docs/en/ingest-management/elastic-agent/elastic-agent-logging.asciidoc

```diff
@@ -0,0 +1,132 @@
+[[elastic-agent-logging-config]]
```

Comment:
Are you also planning to document how to configure and view logs for Fleet-managed Elastic Agents? I think covering the Fleet-managed use case is probably a bit more important than standalone, though arguably the standalone config is harder for users to figure out on their own.

Maybe add a topic about logging for Fleet users that describes:

* How to turn Agent logging off and on (set **Collect Agent logs**  under Create agent policy > Advanced options > Agent monitoring). Looks like it's on by default, but users will want to know where the setting lives and how to change it.
* How to view agent logs (under Agents, click the host name, then click Logs). 

You might want to show that users can select a log level:

![image](https://user-images.githubusercontent.com/14206422/123342897-74704680-d505-11eb-8cc9-b2ab9766b2d9.png)

And that they can view the logs for related programs, like Filebeat and Fleet Server:

![image](https://user-images.githubusercontent.com/14206422/123343349-74247b00-d506-11eb-8f29-aca04e35b037.png)

Reply:
@dedemorton I'm wondering should we go ahead and merge this PR first, before creating the topic for Fleet-managed agents? Currently with 7.14 the elastic_agent logs are not displaying and the team are looking into it.

Reply:
Sure, that sounds good.